recipes/mlgb/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mlgb" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mlgb-{{ version }}.tar.gz
+  sha256: 68b22a5663bcb908b0f02294508c77ae60c527b295b1a0b3b2aeb510a4c23fef
+
+build:
+  skip: True  # [win, py<38]
```

Comment:
This one should be noarch. It is OK to fail on Windows due to missing dependencies in that case.

Reply:
@ocefpaf Okay